src/test/java/org/minions/devfund/damianvp/WhackAMoleTest.java

```diff
@@ -0,0 +1,194 @@
+package org.minions.devfund.damianvp;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Class to test the WhackAMole Class.
+ */
+public class WhackAMoleTest {
+
+    public static final int ATTEMPTS = 50;
+    public static final int GRID_DIMENSION = 10;
+    private WhackAMole whackAMoleGame;
+
+    /**
+     * Instance WhackAMole class in order to start a new Game.
+     */
+    @Before
+    public void startNewGame() {
+        whackAMoleGame = new WhackAMole(ATTEMPTS, GRID_DIMENSION);
+    }
+
+    /**
+     * Test place Mole using valid locations.
+     */
+    @Test
+    public void testPlaceMoleValidPositions() {
```

Comment:
Please write your unit tests like edx
You should have the same quantity of unit tests